src/Aspire.Hosting.AWS/CloudFormation/CloudFormationExtensions.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Amazon.CloudFormation;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.AWS;
+using Aspire.Hosting.AWS.CloudFormation;
+using Aspire.Hosting.Lifecycle;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Aspire.Hosting;
+
+public static class CloudFormationExtensions
+{
+    /// <summary>
+    /// Add a CloudFormation stack for provisioning application resources.
+    /// </summary>
+    /// <param name="builder"></param>
+    /// <param name="stackName">The name of the CloudFormation stack.</param>
+    /// <param name="templatePath">The path to the CloudFormation template that defines the CloudFormation stack.</param>
+    /// <returns></returns>
+    public static IResourceBuilder<ICloudFormationResource> AddAWSCloudFormationProvisioning(this IDistributedApplicationBuilder builder, string stackName, string templatePath)
```

Comment:
```suggestion
    public static IResourceBuilder<ICloudFormationResource> AddCloudFormation(this IDistributedApplicationBuilder builder, string stackName, string templatePath)
```

Reply:
@davidfowl I renamed `AddAWSCloudFormationProvisioning` to `AddAWSCloudFormationTemplate`. I'm thinking I will also add a `AddAWSCloudFormationStack` to essentially import resources from a stack created outside of AppHost. So in the end there will be `AddAWSCloudFormationTemplate` for stacks managed by AppHost and `AddAWSCloudFormationStack` for stacks managed outside of AppHost.